department-of-veterans-affairs/abd-vro
# Force later version of io.netty-handler and fastapi for starlette

## What was the problem?

- [there was a vulnerability found for an out of date io.netty-handler version](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/1113)
- [there was a vulnerability found for an out of date starlette version ](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/865) which was a transitive dependency of Team CC's fastapi


Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/abd-vro/issues/1579

## How does this fix it?[^1]

- makes new constraint to use new io.netty-handler version

## How to test this PR
- Successful SecRel

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
